pdns/recursordist/syncres.cc

```diff
@@ -2992,7 +3003,7 @@ bool SyncRes::doCacheCheck(const DNSName& qname, const DNSName& authname, bool w
       if (!giveNegative)
         res = 0;
       LOG(prefix << qname << ": updating validation state with cache content for " << qname << " to " << cachedState << endl);
```

Comment:
Perhaps we might want to add an EDE status on positive hits as well? But that might be a bit too much?

Reply:
IMO that would be a bit too much